packages/mobile/src/screens/chat-screen/ChatListBlastItem.tsx

```diff
@@ -0,0 +1,75 @@
+import { useCallback } from 'react'
+
+import { useChatBlastAudienceContent } from '@audius/common/hooks'
+import type { ChatBlast } from '@audius/sdk'
+import { css } from '@emotion/native'
+import { TouchableHighlight } from 'react-native'
+
+import {
+  Text,
+  Flex,
+  IconTowerBroadcast,
+  IconUser,
+  useTheme
+} from '@audius/harmony-native'
+import { useNavigation } from 'app/hooks/useNavigation'
+
+import type { AppTabScreenParamList } from '../app-screen'
+
+const messages = {
+  audience: 'audience'
+}
+
+export const ChatListBlastItem = ({ chat }: { chat: ChatBlast }) => {
+  const { spacing } = useTheme()
+  const navigation = useNavigation<AppTabScreenParamList>()
+  const { chat_id: chatId } = chat
+
+  const handlePress = useCallback(() => {
+    navigation.push('Chat', { chatId })
+  }, [navigation, chatId])
+
+  const { audienceCount, contentTitle, chatBlastTitle } =
+    useChatBlastAudienceContent({ chat })
+
+  return (
+    <TouchableHighlight onPress={handlePress}>
+      <Flex
+        ph='xl'
+        pv='l'
+        backgroundColor='white'
+        borderBottom='default'
+        gap='s'
+      >
+        <Flex row gap='s' w='100%' style={css({ overflow: 'hidden' })}>
+          <IconTowerBroadcast
+            width={spacing.xl}
+            height={spacing.xl}
```

Comment:
isn't there a size prop?
maybe not for mobile?

Reply:
seems like no :(